css/properties/-webkit-line-clamp.json

```
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-box-reflect": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-webkit-line-clamp",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": "17"
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "68"
+            },
+            "firefox_android": {
+              "version_added": "68"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": "5"
+            },
+            "safari_ios": {
+              "version_added": "5"
```

We're using iOS versions in our data, rather than the Safari versions.  Safari 5 landed in iOS 4.2 based upon my research.  (It looks like Can I Use stores this data incorrectly -- [Apple stated](https://github.com/mdn/browser-compat-data/issues/2006#issuecomment-457277312) that documentation should be based upon the iOS versions rather than the Safari versions.)

```suggestion
              "version_added": "4.2"
```